tabs/utils/git-auto-conf-cli.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh -e
+
+# Import common utilities
+. ../common-script.sh
+
+# Function to prompt for GitHub configuration
+setup_git_config() {
+    # Prompt for GitHub email
+    printf "Enter your GitHub email address: "
+    read email
+
+    # Prompt for SSH key type
+    printf "Choose your SSH key type:"
+    printf "1. Ed25519 (recommended)"
+    printf "2. RSA (legacy)"
+    printf "Enter your choice (1 or 2): "
```

Comment:
Not like that. This is how it looks with `printf`s:
`Choose your SSH key type:1. Ed25519 (recommended)2. RSA (legacy)Enter your choice (1 or 2): `

```suggestion
    echo "Choose your SSH key type:"
    echo "1. Ed25519 (recommended)"
    echo "2. RSA (legacy)"
    printf "Enter your choice (1 or 2): "
```